tracing-fmt/src/lib.rs

```diff
@@ -206,6 +215,24 @@ where
     }
 }
 
+pub trait NewWriter {
```

Comment:
I took this name from the proposal in #225, but let me know if a different name is desired.

Reply:
We might want to consider renaming this to `MakeWriter`, for consistency with the  `MakeVisitor` trait that will replace `NewVisitor` in  #241 --- see https://github.com/tower-rs/tower/issues/108#issuecomment-430362089 and https://github.com/tokio-rs/tracing/issues/240#issuecomment-518385053 for a discussion of this naming.